src/picotorrent/mainwindow.cpp

```diff
@@ -669,6 +670,15 @@ void MainWindow::onFileAddMagnetLinks()
     dlg->setWindowTitle(i18n("add_magnet_link_s"));
     dlg->open();
 
+    QClipboard* clipboard = QApplication::clipboard();
+    QString text = clipboard->text();
+    if (clipboard->text().startsWith(
```

Comment:
Use the already available `text` variable and put the `if` statement on a single line.